qa/rpc-tests/bip65-cltv-p2p.py

```diff
@@ -43,7 +43,7 @@ def __init__(self):
     def setup_network(self):
         # Must set the blockversion for this test
         self.nodes = start_nodes(self.num_nodes, self.options.tmpdir,
-                                 extra_args=[['-debug', '-whitelist=127.0.0.1', '-blockversion=3']],
+                                 extra_args=[['-debug', '-whitelist=127.0.0.1', '-blockversion=3', '-buriedsfparams=bip65:1351']],
```

Comment:
I have in mind you changed the heights for enforcement to be the same for all tests? So you could revert the patch that makes the test mine additional blocks and reduce `buriedsfparams` accordingly.